libs/components/src/progress-bar/vl-progress-bar.component.cy.ts

```diff
@@ -43,40 +89,82 @@ describe('component vl-progress-bar - default', () => {
                 .contains(step);
         });
     });
+});
 
-    it('should emit vl-click-step event when a step is clicked', () => {
-        cy.createStubForEvent('vl-progress-bar', 'vl-click-step');
-        cy.get('vl-progress-bar').shadow().find('.vl-progress-bar__step:nth-child(2) button').click();
-        cy.get('@vl-click-step').should('have.been.calledWithMatch', { detail: { step: steps[1], number: 2 } });
-        cy.get('@vl-click-step').should('not.have.been.calledWithMatch', { detail: { step: steps[1], number: 1 } });
+describe('vl-progress-bar component - properties default ', () => {
```

Comment:
De naam van de describe begint altijd met de soort test, in dit geval zou het `component vl-progress-bar - properties default` worden.

Reply:
ik heb het veranderd en ik begrijp wat jij wil zegen.

Maar ik denk dat in deze ctx,  de woord `vl-progress-bar` meer belangrijk dan de woord `component` is  dus moet  misschien beter `vl-progress-bar`  in eerste plaats zetten ?